about/README.md

```diff
@@ -12,6 +12,17 @@ turn these digital inspirations into the actual pieces intended for the physical
 The following application materials are included here:
 
  - [Project Statement](project-statement.md) a summary (less than one page) that describes how the work here relates to Red Hat's Open Source Values.
+                            RedHat Project statement Natacha Sochat and Vanessa Sochat
+
+The conceptual nature of this proposal embodies how an open society benefits from all of its members participating in its well-being. ‘The whole can be greater than any one of its parts’, an old but true cliché. This is a core tenet of Open Source values. 
+As a child of immigrants I am very aware of being UpRooted. As an adult, rather than viewing cultural uprooting as a loss, I see the world a fuller, richer condition in which the seemingly chaotic disruption of clashing cultures actually results in the creation of stronger hybrid roots. 
```

Comment:
This is great but you have to remember it's a project proposal. So far I've done most of the work here, and I'm not a "child of immigrants" etc. Do you think it's more appropriate to have individual artist statements, separate from the (shared) project statement (which fits the spirit of the one that I've already written?)

Reply:
Notice there is a document called 'project-statement.md' that links directly from this primary README. That's where the project statement is, I guess you didn't see it?